install/kubernetes/experimental-install.yaml

```diff
@@ -104,15 +104,26 @@ data:
   # container image names
   sidecar-istio-proxy-image: "cilium/istio_proxy"
 
+  # Name of the cluster. Only relevant when building a mesh of clusters.
+  cluster-name: default
+  # Unique ID of the cluster. Must be unique across all conneted clusters and
+  # in the range of 1 and 255. Only relevant when building a mesh of clusters.
+  cluster-id: ""
+
   # Encapsulation mode for communication between nodes
   # Possible values:
   #   - disabled
   #   - vxlan (default)
   #   - geneve
   tunnel: vxlan
+<<<<<<< HEAD
 
   # Name of the cluster. Only relevant when building a mesh of clusters.
   cluster-name: default
+=======
+  # Enables L7 proxy for L7 policy enforcement and visibility
+  enable-l7-proxy: "true"
+>>>>>>> helm/cilium-configmap: added check to deduplicate tunnel and ipam key
```

Comment:
left-overs?

Reply:
Thanks for catching this. I double checked that I had resolved all the conflicts, probably something went wrong in the 2nd push when I removed one of the changes per last comment. I'll fix it.